.dccache

```diff
@@ -0,0 +1 @@
04135594.298,"ab242dbddf5de8f193151081e8a292e0e6248f68c6df16a69c100438ee9f627e"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-details/workflow-details.tsx":[14259,1626804135598.2983,"58d0cf49ac83101bb18a71200fe22b0fe1e9ead22baa67c53c681c30551e7f1f"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-details/workflow-resource-panel.tsx":[375,1626804135598.2983,"84e62d33f2e024684296ac93d6d03b77323409a5c94ee9a310439af82d6cc28a"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx":[5036,1626804135598.2983,"4ac6fb91ab4c5b522a098c9ee70fdbc36b2a9c93c2fc272e222e01fc982b6e73"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-filters/workflow-filters.tsx":[6159,1626804135598.2983,"43b7d553cd1a4be01f01d78e9d4fe6cacb85645d88230ea9fa6fbefd7bbf7735"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-labels/workflow-labels.tsx":[1348,1626804135606.2983,"f1b5798e8d6b5c22490ccc837f405b83103a39f843175558e19ee31986a1d311"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-logs-viewer/full-height-logs-viewer.tsx":[705,1626804135606.2983,"91660297688105e9fa7fd85c2fef7174c6d3f78e7a849a47d42355ffcabcc87e"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx":[6575,1626804135606.2983,"b2ba9380f5482418d15d0179b66f3bd2b37252c75c741054d98b2f9006b0cb64"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx":[18751,1626804135606.2983,"1483df2a9d9a6d7db62d8744e4872f5b6363153ba79a393dffa4a984269974a1"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-panel/workflow-panel.tsx":[1239,1626804135606.2983,"4be1388632cab29ab077f6b45de452b3fcfd6e7432cda7dc9c285f92af4efb7d"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-timeline/workflow-timeline.tsx":[6668,1626804135606.2983,"e5317eae47cc09e974fe50fb86159599e0a95f41a9e303eb177afb967db3ace4"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-yaml-viewer/workflow-yaml-viewer.tsx":[2712,1626804135606.2983,"063b7bfa01df6218c1c03ae30f6404fc45eb2f494f13c2cbe610fc65053d0f45"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflows-list/workflows-list.tsx":[20234,1626804135606.2983,"e8ef1001ff9dcb6e44613f6589c7957cc69e9562b300a0cd092fd6a75ed0a015"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflows-row/workflows-row.tsx":[4632,1626804135610.2983,"a6375beca1daa4ed369d7ecb1050fd885c46bb8283bb1316bc1bdf9e35b7180c"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx":[4572,1626804135610.2983,"984f1f98ee7e86ce7656e4412d1b387f6da5e2fa934b6840711b8ead34ef3d54"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-dag/graph/collapsible-node.ts":[742,1626804135590.298,"ccf6c589b5bb36a04157f724d739e745a8afab7de9b74124442f5d211ce1428b"]}
```

Comment:
I think this along with changes in `go.mod` are some accidental commits, can you remove them please